### DISCUSSION TITLE
It seems like a lot of GGML models are releasing that won't work in ooba or even llama.cpp? ggllm.cpp?

### DISCUSSION THREAD

**Discussion by viperwasp** [oobabooga/text-generation-webui#2771] [General]
It seems like a lot of GGML models are releasing that won't work in ooba or even llama.cpp? 
They are called ggllm.cpp? Do we think these will eventually come to "Ooba" if not what is the recommend best
way to set them up and run them? Any youtube guides or guide in general? Or should I wait for Ooba support?

**Comment by berkut1** [comment]
Do you talk about a new model that support up to 8k context (default llama has 2k)? I heard that KoboldCpp support them and I think ooba will add that too (just why not?:)).

**Comment by viperwasp** [comment]
I'm talking about this model. And ones like it. 
https://huggingface.co/TheBloke/WizardLM-Uncensored-Falcon-40B-GGML
These files are experimental GGML format model files for [Eric Hartford's WizardLM Uncensored Falcon 40B](https://huggingface.co/ehartford/WizardLM-Uncensored-Falcon-40b).

These GGML files will not work in llama.cpp, and at the time of writing they will not work with any UI or library. They cannot be used from Python code.

They can be used with a new fork of llama.cpp that adds Falcon GGML support: [cmp-nc/ggllm.cpp](https://github.com/cmp-nct/ggllm.cpp)

**Reply by berkut1** [reply]
There is also another model https://huggingface.co/TheBloke/minotaur-15B-GGML with 8k context support

**Reply by viperwasp** [reply]
Yes I would love to see these get "Ooba" support. Thanks.

**Reply by syaoran68** [reply]
looking at your link to the wizardLM falcon 40B it says on the first page that its not compatible with the normal llama.. you have to use a specific fork of the llama.cpp code just to get it to run. we'll have to wait until they merge those changes from the fork into the mainline for llama and then for ooba to pull those changes into text-generation-ui
